tcms/xmlrpc/api/forms/testplan.py

```diff
@@ -0,0 +1,35 @@
+from django.forms import BooleanField, CharField, ModelChoiceField
+
+from tcms.management.models import Product, Version
+from tcms.testplans.forms import NewPlanForm
+from tcms.testplans.models import PlanType
+from tcms.xmlrpc.forms import XMLRPCCheckboxInput
+
+
+class BasePlanForm(NewPlanForm):
+    is_active = BooleanField(
+        required=False,
+        widget=XMLRPCCheckboxInput
+    )
+
+
+class NewPlanForm(BasePlanForm):
+    text = CharField()
```

Comment:
This makes the `text` field required and the model specifies this field is not required. So in fact you can drop this field and use the `NewPlanForm` name for the base class instead!